### PR TITLE
Create docker compose for dev and prod environments

### PR DESCRIPTION
**Pull request by ndestrieux**

### CODE REVIEW

File: deployment/dev/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+    PYTHONUNBUFFERED 1
+
+RUN apt-get update \
+    && apt-get -y install curl \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN pip install poetry
+
+COPY ./pyproject.toml ./poetry.lock /app/
```
Comment (kgarbacinski) [inline deployment/dev/Dockerfile]: shouldn't we use the less number of COPY commands?

File: src/conf.py [hunk 2]
```diff
@@ -6,9 +6,10 @@
 class Settings(BaseSettings):
     model_config = SettingsConfigDict(env_file=".env")
```
Comment (kgarbacinski) [inline src/conf.py:7]: it should be uppercased

**Review by kgarbacinski — APPROVED** [review]
no blockers, so approved but please address the comments

File: deployment/dev/Dockerfile (hunk 1 above)
Comment (ndestrieux) [reply deployment/dev/Dockerfile]: Fixed.

**Pull request merged**